client/components/editor.vue

```diff
@@ -306,4 +412,35 @@
     z-index: 2;
     top: 64px;
   }
+
+  .bubble {
```

Comment:
What is this bubble?

Reply:
That's a left-over from the old UI. I forgot to remove it.

Reply:
Oh, you didn't remove this yet. I already merged it.

Can you make a PR removing styles you do not need?